cmd/aspect/root/BUILD.bazel

```diff
@@ -35,12 +35,12 @@ go_library(
         "//cmd/aspect/sync",
         "//cmd/aspect/test",
         "//cmd/aspect/version",
-        "//docs/help/topics",
```

Comment:
sorry these changes are still spurious since copybara isn't configured to match 100%. Have to be manually reverted after the sync